### ISSUE TITLE
BlockQuote 

### ISSUE THREAD

**Issue by elenaverling** [telus/tds-community#214]
## Problem Statement

For large bodies of text, we currently have no styling for pull-out quotes. It's important to have a TDS approved component for consistency, and so the text tag is not labelled as a H2/H1 for accessibility. 

## Design intent

  - Create a TDS styled pull-out quote.
- As a content creator, I want to be able to break up long content with meaningful quotes where necessary, so that my content is easier to read for articles/about pages/blog to the user.
- Best practice from a content perspective: https://docs.google.com/document/d/1kYSBaur2wH69EkUvTSdXUnFbM5qaYsGE5SzoLw8CsF0/edit

## Designs

- Screenshots: attached
![BlockQuote](https://user-images.githubusercontent.com/39060415/57403458-bf089a80-71a7-11e9-9235-8f6262331951.jpg)
- Sketch file: https://invis.io/a/7S5X3D1IJGKU
- InVision links: 
https://telus.invisionapp.com/share/2WFVPUOACPQ#/362279082_BlockQuote

## Design + Dev

- Willing to design solution: Yes (Elena Verling)
- Willing to develop solution: Yes (Andrew Lam)

**Comment by elenaverling** [comment]
This has already been brought to the DPA meeting as part of the blog UI sign off. We're now making it a community component for others to use.

**Comment by varunj90** [comment]
@elenaverling will bring this to DPA today

**Comment by elenaverling** [comment]
I've added BlockQuote to InVision and DSM and written content criteria with Chantelle (content manager). I will need @Christina-Lo or @donnavitan to overlook the Sketch file as I'm not sure if I need to put padding in if there's only one item of text? Let me know!

**Comment by donnavitan** [comment]
Hello @elenaverling, thanks for sharing! Let me know if you have any questions with the recommendations below. And if you'd like to design pair, I'll be available.

**Recommendations**
- Edit best practice bullets to have positive and active language, especially when they are 
-- recommendations and not a hard no or breaking requirement
-- Could include advice on how/when to use
-- Remove "From a marketing perspective...", and "... with exceptions" this is generally good advice for all and people always find exceptions when they need to
- Leave the "don'ts" at the end of the list
- Is it possible that the author may not be required when used as a pull quote
- Provide development considerations in the Github issue, use of a11y and what it will be programmatically? Use of blockquote, cite, etc?
- Where will the hyperlink appear? What style will it be?
- Make components responsive
-- You'll need to build this using Anima
-- You'll need to use some box spacing placeholders
- User lowercase for component name
- Rename to small (≤ 767px) and large (≥ 768px)
- Apply existing layer styles (enables ability for quick changes across components)
- Remove extraneous layer styles
- Remove padding info for desktop/mobile (keep for technical documentation but not for Sketch notes)

**Comment by donnavitan** [comment]
Just adding a note that I'll be design paring with @elenaverling this coming Wednesday. We'll work on making applying these updates. Thanks!

**Comment by donnavitan** [comment]
@elenaverling and @Andrew-K-Lam, design QA on DSM library complete! Looks good. Thank you!

**Comment by donnavitan** [comment]
As per discussion with @elenaverling, @Andrew-K-Lam, and @jraff, what was known as "BlockQuote" will be renamed to "blockquote" in alignment with coding name convention and that it's actually one word. @elenaverling is updating the DSM library and working source files. @Andrew-K-Lam will update his code. Thank y'all!

**Comment by donnavitan** [comment]
Thank you @elenaverling, DSM library components and working source files are updated.

**Comment by varunj90** [comment]
This is now being QA'ed by @charliehay - thanks @charliehay !

**Comment by charliehay** [comment]
Done and Done

**Comment by theetrain** [comment]
Pending final release

- [ ] npm
- [ ] DSM
- [ ] documentation

**Comment by varunj90** [comment]
Docs: https://tds.telus.com/community/index.html#blockquote
DSM: tds_community v.1.0.21
Release notes: https://github.com/telus/tds-community/releases/tag/%40tds%2Fcommunity-blockquote%401.0.0
npm: https://www.npmjs.com/package/@tds/community-blockquote
